src/api/submission/validations.py

```diff
@@ -945,18 +947,27 @@ def validate_percent_values(self):
             percent_soft = ob.get("percent_soft") or 0
             percent_algae = ob.get("percent_algae") or 0
             values = [percent_hard, percent_soft, percent_algae]
-            if self._validate_percent_values(values, "validate_percent_values") == ERROR:
+            if (
```

Comment:
Is using parentheses in these spots really more readable/pythonic?

Reply:
Black formatting